spec/requests/post_request_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET ' do
+    it 'returns http success for /index' do
+      get '/users/1/posts'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'response body includes correct placeholder text for /index' do
+      get '/users/1/posts'
+      expect(response.body).to include('Posts are coming soon!')
+    end
+
+    it 'correct response status for /show' do
+      get '/users/1/posts/1'
+      expect(response).to be_successful
+    end
+
+    it 'response body includes correct placeholder text for show' do
+      get '/users/1/posts/1'
+      expect(response.body).to include('Here is a list of posts for a given user!')
+    end
```

Comment:
- [ ] For the posts' show test, kindly include a test that checks that the show template is rendered.

  ```
  expect(response).to render_template(:show)
  ```